test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/01_file_tailing_test.go

```diff
@@ -98,30 +103,30 @@ func (s *LinuxFakeintakeSuite) TestLinuxLogTailing() {
 func (s *LinuxFakeintakeSuite) LogCollection() {
 	t := s.T()
 	// Create a new log file with permissionn inaccessible to the agent
-	s.Env().RemoteHost.MustExecute("sudo touch /var/log/hello-world.log")
+	s.Env().RemoteHost.MustExecute(fmt.Sprintf("sudo touch %s", logFilePath))
 
 	// Adjust permissions of new log file before log generation
-	output, err := s.Env().RemoteHost.Execute("sudo chmod +r /var/log/hello-world.log && echo true")
+	output, err := s.Env().RemoteHost.Execute(fmt.Sprintf("sudo chmod +r %s && echo true", logFilePath))
 
-	assert.NoError(t, err, "Unable to adjust permissions for the log file '/var/log/hello-world.log'.")
-	assert.Equal(t, "true", strings.TrimSpace(output), "Unable to adjust permissions for the log file '/var/log/hello-world.log'.")
+	assert.NoError(t, err, fmt.Sprintf("Unable to adjust permissions for the log file '%s'.", logFilePath))
```

Comment:
Can you use `assert.NoErrorf` for this one to keep things uniformed

Reply:
Will do 👍